src/OpenTelemetry.AutoInstrumentation.BuildTasks/OpenTelemetry.AutoInstrumentation.BuildTasks.targets

```diff
@@ -0,0 +1,36 @@
+<Project>
+
+    <ItemGroup Label="InstrumentationTargetItems">
```

Comment:
Two follow-up issues:

1. the generation of entries here can be auto-generated from `src/OpenTelemetry.AutoInstrumentation/InstrumentationDefinitions.cs` #2555 
2. move the remaining instrumentation packages from the `Runtime,Managed` package to `InstrumentationTarget` handled by the `BuildTasks` package #2556